lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -483,6 +490,60 @@
             });
         };
 
+        StudentAdmin.prototype.override_problem_score_single = function() {
+            var defaultErrorMessage, fullDefaultErrorMessage, fullSuccessMessage,
+                problemToReset, score, sendData, successMessage, uniqStudentIdentifier,
+                that = this;
+            uniqStudentIdentifier = this.$field_student_select_grade.val();
+            problemToReset = this.$field_problem_select_single.val();
+            score = this.$field_select_score_single.val();
+            if (!uniqStudentIdentifier) {
+                return this.$request_err_grade.text(
+                    gettext('Please enter a student email address or username.')
+                );
+            }
+            if (!problemToReset) {
+                return this.$request_err_grade.text(
+                    gettext('Please enter a problem location.')
+                );
+            }
+            if (!score) {
+                return this.$request_err_grade.text(
+                    gettext('Please enter a score.')
+                );
+            }
+            sendData = {
+                unique_student_identifier: uniqStudentIdentifier,
+                problem_to_reset: problemToReset,
+                score: score
+            };
+            successMessage = gettext("Started task to manually update the score for problem '<%- problem_id %>' and student '<%- student_id %>'. Click the 'Show Task Status' button to see the status of the task.");  // eslint-disable-line max-len
```

Comment:
Sorry to flip-flop, but I just found that this is already possible for ORA problems and we call it "override" both in the UI and the docs. I'd recommend changing "manually update" back to "override."